### DISCUSSION TITLE
File transfer and file copy/paste can't work with network (shared over network) files

### DISCUSSION THREAD

**Discussion by mtelis** [rustdesk/rustdesk#1948] [General]
Subject says it all, I can't download or upload file from network (mapped drive or shared folder). The only way to download network file is to copy it to the remote machine (say, remote desktop) and then download from there.

It doesn't work in reverse direction, either.

Is it possible to resolve this issue?

**Comment by rustdesk** [comment]
i wanna know also. help wanted

**Comment by mtelis** [comment]
Yes, both TeamViewer and Anydesk support network drives and shared files.
In addition, Teamviewer supports Drag&Drop, you can drag and drop files
between the local and remote machines.

**Reply by egpgit** [reply]
Drag and drop files from local to remote rustdesk wolud be great, or copy and paste files

**Comment by DonnEdwards** [comment]
![image](https://github.com/user-attachments/assets/bba10ccd-71cb-4fde-8210-c9c5f94401a9)

See https://github.com/rustdesk/rustdesk/issues/11162 for further details. Bug partly fixed. I am not using hosting, just standard out of the box install.

**Reply by DonnEdwards** [reply]
Self-hosted setup doesn't have this probelm, AFAIK

**Comment by qq147735456** [comment]
Using file transfer tools is too low-level

**Reply by DonnEdwards** [reply]
Too low level for what? Please elaborate

**Comment by rustdesk** [comment]
Close to favor https://github.com/rustdesk/rustdesk/issues/13102

**Comment by rustdesk** [comment]
We accidentally find some clue about this. Will let you test and verify.

**Reply by mtelis** [reply]
Thank you, eagerly waiting

**Comment by fufesou** [comment]
@mtelis @DonnEdwards Could you please

1. Test the portable and the installed versions from the official site https://github.com/rustdesk/rustdesk/releases/tag/1.4.4
3. Test this build when it's available https://github.com/fufesou/rustdesk/releases/tag/test-fs-cm .  Both the portable and installed versions.

**Reply by fufesou** [reply]
https://github.com/fufesou/rustdesk/releases/tag/test-fs-cm is ready.

**Comment by mtelis** [comment]
Checked, it works! Thank you!

**Reply by fufesou** [reply]
Thanks for the test and reply! The portable version works or the test build works?

**Comment by mtelis** [comment]
I tested rustdesk-1.4.4-x86-sciter.exe -- remote machine is running Windows 7.

**Reply by fufesou** [reply]
Thanks! Please also test the official build

https://github.com/rustdesk/rustdesk/releases/tag/1.4.4

**Comment by mtelis** [comment]
Tested and it works like charm. FYI, I uninstalled current version and installed official build to make sure I'm testing correct version.

**Reply by fufesou** [reply]
Thank you for the testing and feedback.

It's great that the official version (https://github.com/rustdesk/rustdesk/releases/tag/1.4.4) is working well for you.

Have a nice weekend!

**Reply by mtelis** [reply]
You as well!

**Reply by fufesou** [reply]
Just checking: have you tested only the portable version?

If you haven't tested the installed version yet, could you please do so?

**Reply by mtelis** [reply]
> Tested and it works like charm. FYI, I uninstalled current version and installed official build to make sure I'm testing correct version.

I tested installed version, see quoted above.
